README.md

```diff
@@ -1,9 +1,111 @@
-# Splunk> Phantom
+[comment]: # "Auto-generated SOAR connector documentation"
+# Talos Intelligence
 
-Welcome to the open-source repository for Splunk> Phantom's talosintelligencev2 App.
+Publisher: Splunk Community  
```

Comment:
I think we'd be changing this to be Splunk / Splunk supported info

Reply:
captured all of this in https://splunk.atlassian.net/browse/PAPP-34667